### PR TITLE
US-6.4: Implement my account portlet

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml [hunk 1]
```diff
@@ -245,8 +245,8 @@
         <access-permissions>*:/platform/users</access-permissions>
         <portlet-application>
           <portlet>
-            <application-ref>platformNavigation</application-ref>
-            <portlet-ref>UISpaceNavigationPortlet</portlet-ref>
+            <application-ref>digital-workplace</application-ref>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml]: To be reverted for now

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/myAccount/components/MyAccountApp.vue [hunk 2]
```diff
@@ -3,7 +3,38 @@
     id="digital-workplace-myAccount"
     class="VuetifyApp"
     flat>
-    Profile navigation works!
+    <v-container
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/myAccount/components/MyAccountApp.vue]: The my account links should be active after corresponding navigation nodes and pages created

Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/myAccount/components/MyAccountApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml (hunk 1 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml [hunk 3]
```diff
@@ -11,5 +11,28 @@
...
+      <uri>profile</uri>
+      <name>profile</name>
+      <label>#{portal.dw.profile}</label>
+      <visibility>SYSTEM</visibility>
+      <page-reference>portal::dw::profile</page-reference>
+    </node>
+
+    <node>
+      <name>achievements</name>
+      <label>Gamification Achievements</label>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml]: To be internationalized also

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml [hunk 4]
```diff
@@ -11,5 +11,28 @@
...
+      <visibility>SYSTEM</visibility>
+      <page-reference>portal::dw::achievements</page-reference>
+    </node>
+
+    <node>
+      <name>wallet</name>
+      <label>Wallet</label>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml]: To be internationalized also

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml (hunk 3 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml (hunk 4 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/navigation.xml]: Done!

**Pull request merged**